xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/layouts/table/LayoutTableHeaderNames.vue

```diff
@@ -142,17 +163,53 @@ export default {
     },
 
     reorderProperty (e) {
-      // As the draggable plugin is taking in account every child it has for d&d
-      // and there is the select-entry-all component as first child
-      // we need to substract 1 to the indexes that the draggable plugin handles
-      // so that it matches the true property order
-      // When selection is disabled (and the select-entry-all component hidden)
-      // we don't need to readjust the offset of the indexes
-      // as Vue handily creates a html comment where the component shoud be
-      // So that it does not messes up with indexes
+      /*
+        As the draggable plugin is taking in account every child it has for d&d
+        and there is the select-entry-all component as first child
+        we need to substract 1 to the indexes that the draggable plugin handles
+        so that it matches the true property order
+        When selection is disabled (and the select-entry-all component hidden)
+        we don't need to readjust the offset of the indexes
+        as Vue handily creates a html comment where the component shoud be
+        So that it does not messes up with indexes
+      */
```

Comment:
The previous style was OK. We use multi-line comment syntax only for documenting functions / methods and classes. For in-line comments we use the single line syntax.